tests/spec/w3c/headers-spec.js

```diff
@@ -103,11 +110,10 @@ describe("W3C — Headers", () => {
       };
       Object.assign(ops.config, newProps);
       const doc = await makeRSDoc(ops);
-      expect($("dt:contains('Editors:')", doc).length).toEqual(1);
-      expect($("dt:contains('Editor:')", doc).length).toEqual(0);
-      const $dd = $("dt:contains('Editors:')", doc).next("dd");
-      expect($dd.text()).toEqual("NAME1");
-      expect($dd.next("dd").text()).toEqual("NAME2");
+      expect(contains(doc, "dt", "Editors:").length).toBe(1);
+      expect(contains(doc, "dt", "Editor:").length).toBe(0);
+      expect(doc.querySelectorAll("dd")[5].textContent).toBe("NAME1");
```

Comment:
I think this line is to check `NAME1` is exactly after `Editors`, so probably using `nextElementSibling` is better.